### ISSUE TITLE
Footnotes not rendered properly in tables

### ISSUE THREAD

**Issue by rxian** [MSzturc/obsidian-advanced-slides#55]
When a footnote appears in a table, it seems to push texts that come after it onto a new line.

```markdown
| Table     | Col 1 | Col 2 | Col 3 |
| --------- | ----- | ----- | ----- |
| Footnotes | [^1]  | [^2]  | [^3]  | 

[^1]: fn1
[^2]: fn2
[^3]: fn3
```

gets

<img width="532" alt="Screen Shot 2022-02-21 at 12 46 50 PM" src="https://user-images.githubusercontent.com/8406802/155011824-a0c631d7-a65a-491d-8938-f801a17dce54.png">

Sorry for spamming you with issues; I really like this plug-in and it has made my weekly presentation much easier to prepare for. Thank you for your hard work, and I hope they can be helpful in improving this plug-in!

**Comment by rxian** [comment]
This is caused by `\n` appended in [footnote processor](https://github.com/MSzturc/obsidian-advanced-slides/blob/19daa5b20a42fafebbbd31bf720289f5c0ccacd7/src/processors/footNoteProcessor.ts#L54). I guess this to properly handle when multiple footnotes are appearing next to each other, comma should be inserted (as done in latex) instead of new line?

**Comment by MSzturc** [comment]
will be fixed with 1.7.0
